### ISSUE TITLE
Notifications doesn't work

### ISSUE THREAD

**Issue by enoch85** [nextcloud/vm#2159]
### Steps To Reproduce

![image](https://user-images.githubusercontent.com/4511254/144717184-504cc7b3-102d-4450-92b1-bcec9a4c82b3.png)

### Expected Result

.

### Actual Result

.

### Build Version

23.0.0

### Environment

By downloading the VM

**Comment by enoch85** [comment]
cc @szaimen if you have time.

**Comment by szaimen** [comment]
Sounds like thats also due to https://github.com/nextcloud/server/issues/30094

**Comment by szaimen** [comment]
(the user is probably not completely removed and gets still listed in the user:list output, no?

**Comment by enoch85** [comment]
Nope, not listed in that output.

**Comment by szaimen** [comment]
Can you verbose the output on that machine?
so e.g. set -x or bash -x ?

**Comment by enoch85** [comment]
```
+ notify_admin_gui testsubject 'does this work?'
+ local NC_USERS
+ local user
+ local admin
+ is_app_enabled notifications
+ nextcloud_occ app:list
+ check_command sudo -u www-data php /var/www/nextcloud/occ app:list
+ sudo -u www-data php /var/www/nextcloud/occ app:list
+ awk '{print$2}'
+ grep -q '^notifications$'
+ sed '/Disabled/,$d'
+ tr -d :
+ sed '/^$/d'
+ return 0
+ print_text_in_color '\e[0;96m' 'Posting notification to users that are admins, this might take a while...'
+ printf '%b%s%b\n' '\e[0;96m' 'Posting notification to users that are admins, this might take a while...' '\e[0m'
Posting notification to users that are admins, this might take a while...
+ send_mail testsubject 'does this work?'
+ local RECIPIENT
+ '[' -f /etc/msmtprc ']'
+ return 1
+ '[' -z '' ']'
++ nextcloud_occ_no_check user:list
++ sudo -u www-data php /var/www/nextcloud/occ user:list
++ sed 's|^  - ||g'
++ sed 's|:.*||'
+ NC_USERS=admin
+ mapfile -t NC_USERS
+ for user in "${NC_USERS[@]}"
+ cut -d - -f2
+ nextcloud_occ_no_check user:info admin
+ sudo -u www-data php /var/www/nextcloud/occ user:info admin
+ grep -x -q ' admin'

In OC_Helper.php line 496:
                                 
  [OCP\Files\NotFoundException]  
                                 

user:info [--output [OUTPUT]] [--] <user>
```

**Comment by enoch85** [comment]
So `nextcloud_occ_no_check user:info admin` doesn't work here it seems.

@nickvergessen any idea?

**Comment by enoch85** [comment]
This is still an issue.

**Comment by neilrooney** [comment]
Having exactly the same issues since updating to NC v23.

**Comment by enoch85** [comment]
This is a bug in Nextcloud. Let's see what 23.0.1 brings.

**Comment by enoch85** [comment]
This is now fixed for 23.0.1
